logging/request_context_logger.go

```diff
@@ -0,0 +1,70 @@
+package logging
+
+import (
+	"github.com/giantswarm/request-context"
+)
+
+// NewRequestContextLogger returns a RequestContextLogger, given a Config.
+func NewRequestContextLogger(config Config) RequestContextLogger {
+	requestContextConfig := requestcontext.LoggerConfig{
+		Name:                config.Name,
+		Level:               config.LogLevel,
+		Color:               config.Color,
+		IncludeNameInFormat: false,
+	}
+
+	rcl := RequestContextLogger{
+		logger: requestcontext.MustGetLogger(requestContextConfig),
+	}
+
+	return rcl
+}
+
+// RequestContextLogger is a Logger based on the request-context library.
+type RequestContextLogger struct {
+	logger requestcontext.Logger
+}
+
+// contextToCtx takes a logging.Context, and returns a requestcontext.Ctx.
+func contextToCtx(context Context) requestcontext.Ctx {
```

Comment:
requestcontext has `Ctx`, which I didn't want to expose. So, we have `logging.Context`, which we map to `Ctx`.